### PR TITLE
#175494111 creating sign up

### PR DESCRIPTION
**Pull request by Sandrine21**
#### What does this PR do?
sign up page.

#### Description of Task to be completed?

the user will be able to sign up using email or social media.

#### How should this be manually tested?

first, clone it.
then run flutter pub get to install dependencies.
after run flutter run to run it.

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories?

https://www.pivotaltracker.com/story/show/175494111

#### Screenshots (if appropriate)

Screenshot_20201123-120029

#### Questions:
N/A

### CODE REVIEW

**Review by Rita-12397 — APPROVED** [review]

**Review by Jajabenit250 — CHANGES_REQUESTED** [review]
this Pull Request is empty it has no change

**Review by mujohn26 — COMMENTED** [review]
Hey @Sandrine21 This PR doesn't have any changes, actually what did you worked on ?

**Review by dianahub250 — APPROVED** [review]
Well done Sandrine you can continue

File: lib/model/signup.model.dart [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class Info {
+  String name;
+  String email;
+  String password;
+  String confirmp;
```
Comment (mujohn26) [inline lib/model/signup.model.dart]: Please provide a descriptive  variable name.

**Review by mujohn26 — APPROVED** [review]
Looks good to me.

**Pull request merged**